src/lotto/domain/LottoGame.java

```diff
@@ -9,14 +9,15 @@
 
     public void run() {
         int paidAmount = inputView.getPaidAmount();
-        LottoBundle lottoBundle = new LottoBundle(paidAmount);
-        lottoBundle.issueLottoTicketsWithRandomNumbers();
+        LottoBundle lottoBundle = LottoBundle.createByCashValue(paidAmount);
+        lottoBundle.fillWithRandomLottoTickets();
 
         outputView.printPurchaseResult(lottoBundle);
 
-        int[] winningNumber = inputView.getWinningNumber();
-        LottoTicket winningTicket = new LottoTicket(winningNumber);
-        LottoResult lottoResult = new LottoResult(lottoBundle, winningTicket);
+        int[] winningNumberInput = inputView.getWinningNumber();
+        int bonusNumberInput = inputView.getBonusNumber();
+        WinningNumber winningNumber = LottoFactory.selectWinningNumber(winningNumberInput, bonusNumberInput);
+        LottoResult lottoResult = LottoResult.of(lottoBundle, winningNumber);
```

Comment:
```suggestion
        WinningNumber winningNumber = LottoFactory.selectWinningNumber(inputView.getWinningNumber(), inputView.getBonusNumber());
        LottoResult lottoResult = LottoResult.of(lottoBundle, winningNumber);
```
이렇게 하는 거랑 느낌도 비교해 보세요.